plugins/com.google.cloud.tools.eclipse.appengine.newproject/plugin.properties

```diff
@@ -0,0 +1,5 @@
+google = Google
+Bundle-Vendor = Google Inc.
+Bundle-Name = Wizards for Google Cloud Platform Apps
```

Comment:
This Bundle-Name is at odds with the much more constrained bundle symbolic name (`…appengine…`).